pkg/virt-handler/isolation/isolation.go

```diff
@@ -53,9 +54,9 @@ import (
 type PodIsolationDetector interface {
 	// Detect takes a vm, looks up a socket based the VM and detects pid, cgroups and namespaces of the owner of that socket.
 	// It returns an IsolationResult containing all isolation information
-	Detect(vm *v1.VirtualMachineInstance) (*IsolationResult, error)
+	Detect(vm *v1.VirtualMachineInstance) (IsolationResult, error)
 
-	DetectForSocket(vm *v1.VirtualMachineInstance, socket string) (*IsolationResult, error)
+	DetectForSocket(vm *v1.VirtualMachineInstance, socket string) (IsolationResult, error)
```

Comment:
Where does this socket come from? Would you please add a doc string?

Reply:
I am not adding the socket here, so not sure I should document it either. The patch merely introduced an interface here for test purposes, no changes to socket.

Reply:
Ack